### PR TITLE
test: add more tests for creds cache

### PR DESCRIPTION
**Pull request by genryxy**
Part of #87 
Added more tests for credentials cache

### CODE REVIEW

File: src/test/java/com/artipie/YamlSettingsTest.java [hunk 1]
```diff
@@ -175,6 +175,33 @@ void returnsCredentials(@TempDir final Path tmp) throws IOException {
...
+            new StringContains("size=1")
+        );
+        MatcherAssert.assertThat(
+            settings.credentials()
+                .toCompletableFuture().join()
+                .list()
+                .toCompletableFuture().join()
+                .size(),
+            new IsEqual<>(1)
+        );
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/YamlSettingsTest.java:202]: Please, add mismatch description to this assertion

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy thanks, just one minor comment

**Comment by genryxy** [comment]
@olenagerasimova thanks, added

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks

**Comment by genryxy** [comment]
@g4s8 could you please check/merge

**Pull request merged**